ballerina/build.gradle

```diff
@@ -1096,12 +1095,12 @@ task testExamples() {
                 //TODO: Need to verify with windows
                 exec {
                     workingDir "${buildDir}/TestProject"
-                    commandLine 'cmd', '/c', "${distPath}/bin/ballerina.bat", 'build', '--experimental', "${bbe}"
+                    commandLine 'cmd', '/c', "${distPath}/bin/ballerina.bat", 'build', '--experimental', "${bbe} --b7a.home=${distPath}/jballerina-tools-${ballerinaVersion}"
                 }
             } else {
                 exec {
                     workingDir "${baseBuildPath}/TestProject"
-                    commandLine 'sh', '-c', "../jballerina-tools-${ballerinaVersion}/bin/ballerina build --experimental ${bbe}"
+                    commandLine 'sh', '-c', "${baseBuildPath}/jballerina-tools-${ballerinaVersion}/bin/ballerina build --experimental ${bbe} --b7a.home=${baseBuildPath}/jballerina-tools-${ballerinaVersion}"
```

Comment:
shall we use `${distPath}/bin/ballerina` here as well?

Reply:
Yes. Fixed all the places with https://github.com/ballerina-platform/ballerina-distribution/pull/551/commits/f51de3066bb598517eb60177e885880c11be382e